dotnet/src/Connectors/Connectors.HuggingFace.UnitTests/ImageToText/HuggingFaceImageToTextTests.cs

```diff
@@ -0,0 +1,218 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Connectors.HuggingFace;
+using Microsoft.SemanticKernel.Connectors.HuggingFace.ImageToText;
+using Microsoft.SemanticKernel.ImageToText;
+using Xunit;
+
+namespace SemanticKernel.Connectors.HuggingFace.UnitTests;
+
+/// <summary>
+/// Unit tests for <see cref="HuggingFaceTextGenerationService"/> class.
+/// </summary>
+public sealed class HuggingFaceImageToTextTests : IDisposable
+{
+    private readonly HttpMessageHandlerStub _messageHandlerStub;
+    private readonly HttpClient _httpClient;
+    private readonly ImageContent _imageContentInput;
+    public HuggingFaceImageToTextTests()
```

Comment:
Small spacing improvement :)
```suggestion

    public HuggingFaceImageToTextTests()
```